src/screenshot.cpp

```diff
@@ -121,6 +121,39 @@ void Screenshot::savePathScreenshot(const QString &path)
     m_window->savePath(path);
 }
 
+void Screenshot::multioptionalScreenshot(int areaOption, QString savePath){
+    initUI();
+    this->show();
+    if(areaOption == 1){
+        m_window->fullScreenshot(savePath);
+    }
+    else if(areaOption == 2){
+        m_window->topWindow(savePath);
+    }
+}
+
+void Screenshot::multioptionalScreenshot(int delay, int areaOption, QString savePath){
+    QString summary = QString(tr("Deepin Screenshot will start after %1 seconds").arg(delay));
```

Comment:
why not just use `delayScreenshot`?

Reply:
Because I dont't want to delete those duplicate codes before I finishing all the changes. After I make it can disable the notify in ```screenshotWithOptions```, I'd delete them